packages/devtools/test/composite.test.ts

```diff
@@ -739,21 +738,12 @@ describe('composite', () => {
   })
 
   test('Relations support', async () => {
-    const postComposite = await Composite.create({ ceramic, schema: postSchema })
-    const postID = postComposite.modelIDs[0]
-    expect(postID).toBeDefined()
-
-    const postAndCommentComposite = await Composite.create({
-      ceramic,
-      schema: createCommentSchemaWithPost(postID),
-    })
-    const commentID = postAndCommentComposite.modelIDs.find((id) => id !== postID)
-    expect(commentID).toBeDefined()
+    const composite = await Composite.create({ ceramic, schema: postSchema })
+    expect(composite.modelIDs).toHaveLength(2)
```

Comment:
This needs a comment, since post schema actually has 2 models.

Reply:
What would the comment be please? That it's expected to have 2 models?

Reply:
Yep, that post schema includes comments. Or rename it to `postAndCommentSchema`